common/changes/resizegroup-optimize-shrink_2017-05-26-21-04.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Reduce the number of render calls when reducing the width of a ResizeGroup",
```

Comment:
I think the guideline is to say state first the Component affected and then the description.

ResizeGroup: Reduce the number of render calls when reducing the width of a ResizeGroup